content/docs-v1/multiformat-output.md

```diff
@@ -350,4 +490,12 @@ Temporarily skipping the build process in this way can save a lot of time, thoug
 
 {{< q-class "box warning" >}}
 - Remember that any time you run `quire build` any changes you made to the `_site/pdf.html` and `_site/pdf.css` will be erased.
-{{< /q-class >}}
\ No newline at end of file
+{{< /q-class >}}
+
+### PDF Accessibility
+
+Information to come ...
```

Comment:
We have used TK in other parts of the docs, which we may want to rethink. And I would like to keep either "TK" or "information to come" so we remember it needs to go there (in addition to an issue in the docs repo), but also so the community knows there's more helpful information on the way.